ui/job-view/pushes/Push.jsx

```diff
@@ -69,15 +69,22 @@ class Push extends React.PureComponent {
     window.removeEventListener(thEvents.applyNewJobs, this.handleApplyNewJobs);
     window.removeEventListener('hashchange', this.handleUrlChanges);
   }
-
+  
   getJobCount(jobList) {
+    function filterCommit(job) {
```

Comment:
This could be simplified and doesn't really need to be a separate function.  You can pass an arrow function directly into the ``filter`` function.  I don't want to give you the exact code, but the syntax would look very much like this example from MDN:  ``elements.map(element => element.length)``
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions